README.md

```diff
@@ -512,3 +512,7 @@ _See code: [src/commands/single.ts](https://github.com/oclif/oclif/blob/v1.3.8/s
 * [@oclif/errors](https://github.com/oclif/errors) - Renders and logs errors from commands.
 * [@oclif/cli-ux](https://github.com/oclif/cli-ux) - Library for common CLI UI utilities.
 * [@oclif/test](https://github.com/oclif/test) - Test helper for oclif.
+
+# 📣 Feedback
+
+If you have any suggestions or just want to let us know what you think of oclif, send us a message at <heroku-cli@salesforce.com>
```

Comment:
I'm confused, is it `heroku-cli` or `heroku-cli-squad` as mentioned here https://github.com/oclif/oclif/pull/39#issuecomment-373585087 ? 😆

Reply:
Oops we have two email addresses! We're going to use heroku-cli@salesforce.com